src/main/java/lotto/service/LottoMachine.java

```diff
@@ -0,0 +1,34 @@
+package lotto.service;
+
+import lotto.model.LottoGenerator;
+import lotto.model.LottoNumbers;
+import lotto.model.LottoPrizeRanks;
+import lotto.model.LottoQuantityChecker;
+import lotto.model.LottoStatics;
+import lotto.model.RandomNumberGenerator;
+import lotto.view.InputView;
+import lotto.view.ResultView;
+
+import java.util.List;
+
+public class LottoMachine {
+    public void pay() {
+        String price = InputView.inputPrice();
```

Comment:
꼼꼼하게 피드백 잘 반영해주셨네요 👍

로또를 구입하기 위한 금액에 대한 검증부를 잘 분리해주셔서 InputView 에서 처리되도록 변경 해주셨는데요, 혹시 아래와 같은 흐름은 어떻게 생각하시나요?

```suggestion
        LottoMoney price = new LottoMoney(InputView.inputPrice());
```
네이밍은 그냥 예시입니다 😂